pkg/catalog/manager/manager.go

```diff
@@ -63,6 +67,22 @@ func (m *Manager) GetCatalogs() ([]v3.Catalog, error) {
 	return list.Items, nil
 }
 
+func (m *Manager) GetProjectCatalogs() ([]v3.ProjectCatalog, error) {
+	list, err := m.projectCatalogClient.List(metav1.ListOptions{})
```

Comment:
Is there a reason we make a direct list call to api instead of using cache? Using cache is strongly recommended to reduce the number of direct api hits